### DISCUSSION TITLE
Not able to use Random Forest and Gradient Boosting

### DISCUSSION THREAD

**Discussion by SharmaLovely** [sebp/scikit-survival#486] [General]
The sample code is giving error for both random forest and gradient boosting. 

TypeError: _parallel_build_trees() got an unexpected keyword argument 'missing_values_in_feature_mask'

TypeError: Can't instantiate abstract class GradientBoostingSurvivalAnalysis with abstract method _validate_y

**Comment by sebp** [comment]
Looks like you are using an unsupported version of scikit-learn.

Can you please execute the following code and post the output

import sksurv; sksurv.show_versions()

**Reply by SharmaLovely** [reply]
It helped, thank you.
